File: LeetCode/2037使每位学生都有座位的最小移动次数.cpp
int cmp(const void *a, const void *b) {
    return *(int *)a - *(int *)b;
}

int minMovesToSeat(int* seats, int seatsSize, int* students, int studentsSize){
    int ans = 0;
    qsort(seats, seatsSize, sizeof(int), cmp);
    qsort(students, studentsSize, sizeof(int), cmp);
    for (int i = 0; i < seatsSize; i++) {
        ans += abs(students[i] - seats[i]);
    }
    return ans;
}

File: LeetCode/2037使每位学生都有座位的最小移动次数.go
func minMovesToSeat(seats []int, students []int) int {
	sort.Sort(sort.IntSlice(seats))
	sort.Sort(sort.IntSlice(students))
	var ans int = 0
	for i := 0; i < len(seats); i++ {
		ans += int(math.Abs(float64(seats[i]) - float64(students[i])))
	}
	return ans
}